app/services/events/create_service.rb

```diff
@@ -36,5 +37,22 @@ def call
     private
 
     attr_reader :organization, :params, :timestamp, :metadata
+
+    def produce_kafka_event(event)
+      return if ENV['LAGO_KAFKA_BOOTSTRAP_SERVERS'].blank?
+
+      Karafka.producer.produce_sync(
```

Comment:
:wave: you may want to go with `produce_async`. Sync production is in general slow (3-7 ms per message) and async is really reliable and fast. By default it dispatches messages every 5 milliseconds in the background.